StatsigInjections/StatsigInjections/StatsigInjections.swift

```diff
@@ -70,7 +70,7 @@ public final class StatsigFeatureFlagsProvider: NSObject, FeatureFlagsProtocol {
             initializationState = .initializedRemoteLoaded
             completion()
         } else {
-            Statsig.start(sdkKey: apiKey, user: StatsigUser(userID: userId), options: StatsigOptions(
+            Statsig.start(sdkKey: apiKey, user: StatsigUser(userID: userId, customIDs: ["isNativeApp":"true"]), options: StatsigOptions(
```

Comment:
tried this with a bool value, but Statsig seems to only support Strings in the dashboard